### ISSUE TITLE
PackedIntegerArray: Invalid length given for storage, got: 320 but expected: 342

### ISSUE THREAD

**Issue by lhns** [LolHens/dynmap-fabric#9]
```
bitsperblock = 320
statelist.length = 320
```

```
[21:07:49] [Dynmap Render Thread/FATAL]: [Dynmap] Exception during render job: world=world, map=org.dynmap.hdmap.HDMap@442affef
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: java.lang.RuntimeException: Invalid length given for storage, got: 320 but expected: 342
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at net.minecraft.class_3508.<init>(class_3508.java:111)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at org.dynmap.fabric_1_16_1.ChunkSnapshot.<init>(ChunkSnapshot.java:186)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at org.dynmap.fabric_1_16_1.FabricMapChunkCache.prepChunkSnapshot(FabricMapChunkCache.java:1000)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at org.dynmap.fabric_1_16_1.FabricMapChunkCache.readChunks(FabricMapChunkCache.java:1151)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at org.dynmap.fabric_1_16_1.FabricServer.createMapChunkCache(FabricServer.java:418)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at org.dynmap.MapManager$FullWorldRenderState.processTile(MapManager.java:761)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at org.dynmap.MapManager$FullWorldRenderState.run(MapManager.java:720)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$1.run(MapManager.java:229)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$2.run(MapManager.java:245)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[21:07:49] [Dynmap Render Thread/INFO]: [STDERR]: 	at java.lang.Thread.run(Thread.java:745)
```

**Comment by lhns** [comment]
~~I built a workaround for this in 230ed831f59449fe717c0fef15be05fb81bdcd7c that expands the statelist if it doesn't match expected length but I currently don't know if it's the right thing to do.~~
~~It seems to work fine~~ but for now it issues a warning in the console.
EDIT: Replaced by #22 and #23

For continued discussion see https://github.com/LolHens/dynmap-fabric/issues/19#issuecomment-674417191
TLDR: this is not a bug of the fabric port but affects all platforms on 1.16 (forge, spigot, fabric)

**Comment by lhns** [comment]
properly fixed in 0.12.2
